examples/js/e-commerce-umd/src/routing.ts

```diff
@@ -1,12 +1,13 @@
 /* eslint complexity: off */
 
-import type { UiState } from 'instantsearch.js';
 import {
   getFallbackHitsPerPageRoutingValue,
   getFallbackSortByRoutingValue,
   getFallbackRatingsRoutingValue,
 } from './widgets';
 
+import type { UiState } from 'instantsearch.js';
```

Comment:
Does this come after local imports because it's a type import ?

Reply:
Yep, here's the order:

```js
groups: [
  'builtin',
  'external',
  'parent',
  'sibling',
  'index',
  'type',
],
```